go/client/cmd_signup.go

```diff
@@ -422,6 +436,18 @@ func (s *CmdSignup) postInviteRequest() (err error) {
 	return
 }
 
+func (s *CmdSignup) postCodeRequest() error {
```

Comment:
I wouldn't call this `postCodeRequest`, since `post` implies an HTTP `POST`, which usually implies mutating server-side data.  I would call it `requestInvitationCode` or some such.